src/common/ceph_crypto.h

```diff
@@ -97,25 +116,25 @@ namespace ceph {
 	param.type = siBuffer;
 	param.data = NULL;
 	param.len = 0;
-	ctx = PK11_CreateContextBySymKey(cktype, CKA_SIGN, symkey, &param);
+	ctx.reset(PK11_CreateContextBySymKey(cktype, CKA_SIGN, symkey, &param));
```

Comment:
i think this change also fixes a leak of `PK11Context`. as `HMAC` does not call `PK11_DestroyContext(ctx)` before this change.